engine/src/cmd/movable.cpp

```diff
@@ -252,6 +232,13 @@ void Movable::Rotate( const Vector &axis )
 
 Vector Movable::ResolveForces( const Transformation &trans, const Matrix &transmat )
 {
+    static float maxplayerrotationrate = GameConfig::GetVariable( "physics", "maxplayerrot", 24);
+    static float maxnonplayerrotationrate = GameConfig::GetVariable( "physics", "maxNPCrot", 360);
+    static float warpstretchcutoff = GameConfig::GetVariable( "graphics", "warp_stretch_cutoff", 500000) * GameConfig::GetVariable( "physics", "game_speed", 1);
+    static float warpstretchoutcutoff = GameConfig::GetVariable( "graphics", "warp_stretch_decel_cutoff", 500000) * GameConfig::GetVariable( "physics", "game_speed", 1);
+    static float sec = GameConfig::GetVariable( "graphics", "insys_jump_ani_second_ahead", 4) / (GameConfig::GetVariable( "physics", "game_speed", 1) * GameConfig::GetVariable( "physics", "game_accel", 1));
+    static float endsec = GameConfig::GetVariable( "graphics", "insys_jump_ani_second_ahead_end",0.03f) /( GameConfig::GetVariable( "physics", "game_speed",1.0f) *GameConfig::GetVariable( "physics", "game_accel", 1.0f) );
```

Comment:
NOTE: These are the kinds of things that I'd like to see be properties of the class and be initialized as part of the class being initialized. They can be static class members and be initialized the first time the class is initialized for all instances if desired.

```c++
class X
{
protected:
    static bool initialized = false;  // control variable for initialization logic
    static float endsec = 0.0;  // set default value
    ...
}


X::X()
{
   if (!X.initialized) {
       // Note - ideally this would be `config.GetVariable()` where `config` could be passed as an argument or be one of the sole/few global variables we have in the system
       X.endsec = GameConfig::GetVariable(...);
       X.initialized = true;
   }
}
```

I would like to see us move to this style as it would greatly reduce the number of times we have to load configuration data. It would also enable us to easily add a `loadConfig()` function that could be called whenever we needed to load or reload the configuration for the entire class.

Reply:
I like that pattern. I will try implementing at least part of it. The global `config` variable I may leave for later.

Reply:
yes that's fine. As a first step it'd be good to move to this pattern. Once all the code is on this pattern we can look at moving to the global config object too.

Reply:
Done.